app/containers/App/index.js

```diff
@@ -28,7 +28,7 @@ export default function App() {
       <Header />
       <Switch>
         <Route exact path="/" component={HomePage} />
-        <Route exact path="/profile" component={Profile} />
+        <Route exact path="/users/:id" component={Profile} />
```

Comment:
Let's do separate route for current user edit profile screen.
As we discussed, let's separate profile form from profile view screen into separate components. Profile should be just displaying data without input fields.